doc/maintenance.rst

```diff
@@ -298,6 +298,12 @@ At this point, we need to upload the binaries to GitHub and link them to the tag
 Build and deploy the documentation
 ----------------------------------
 
+Before building the documentation, make sure that the following LaTeX
+dependencies are installed on your system:
+    * dvipng
```

Comment:
To make life of maintainers easier, could you add :

- the links to the web page of each tool (ex: https://ctan.org/pkg/dvipng)

Also, it would be awesome to add (if not too much work):

- the commands to check that they are installed
- the commands to download and install them (for dvipng, it'd be something like this I think:

```
$ wget https://mirrors.ctan.org/dviware/dvipng.zip
$ unzip dvipng.zip
$ cd dvipng
$ ./configure
$ make
$ make install
```

Reply:
Yea sure I think it's a good addition thanks!